sorc/ncep_post.fd/CLDRAD.f

```diff
@@ -5311,26 +5220,6 @@ SUBROUTINE CLDRAD
          endif
       ENDIF
       
-!! ADD DUST WET DEPOSITION FLUXES (kg/m2/sec)
-!      IF (IGET(662)>0) THEN
```

Comment:
@zhanglikate Do you think variable duwt can be cleaned up for the same reason above?

Reply:
@WenMeng-NOAA No. As I mentioned above, we do need to output duwt using 662.